lightly/transforms/models/smog_transform.py

```diff
@@ -0,0 +1,140 @@
+from torch import Tensor
+from lightly.transforms.multi_view_transform import MultiViewTransform
+from lightly.transforms.utils import IMAGENET_NORMALIZE
+from lightly.transforms.solarize import RandomSolarization
+from lightly.transforms.gaussian_blur import GaussianBlur
+from typing import Optional, Tuple, Union
+from PIL.Image import Image
+import torchvision.transforms as T
+
+
+class SMoGTransform(MultiViewTransform):
+    """Implements the transformations for SMoG.
+
+    Attributes:
+        crop_sizes:
+            Size of the input image in pixels for each crop category.
+        crop_counts:
+            Number of crops for each crop category.
+        crop_min_scales:
+            Min scales for each crop category.
+        crop_max_scales:
+            Max_scales for each crop category.
+        gaussian_blur_probs:
+            Probability of Gaussian blur for each crop category.
+        gaussian_blur_kernel_sizes:
+            Deprecated values in favour of sigmas.
+        gaussian_blur_sigmas:
+            Tuple of min and max value from which the std of the gaussian kernel is sampled.
+        solarize_probs:
+            Probability of solarization for each crop category.
+        hf_prob:
+            Probability that horizontal flip is applied.
+        cj_prob:
+            Probability that color jitter is applied.
+        cj_strength:
+            Strength of the color jitter.
+        random_gray_scale:
+            Probability of conversion to grayscale.
+        normalize:
+            Dictionary with 'mean' and 'std' for torchvision.transforms.Normalize.
+
+    """
+
+    def __init__(
+        self,
+        crop_sizes: Tuple[int] = (224, 96),
```

Comment:
```suggestion
        crop_sizes: Tuple[int, int] = (224, 96),
```

Similar for the tuples below.